src/mock_etcd.rs

```diff
@@ -128,22 +155,129 @@ impl MockEtcd {
         kvs
     }
 
+    /// Send watch response for a specific watch id to etcd client
+    async fn send_watch_response_with_watch_id(
+        watch_response_sender: Arc<RwLock<DuplexSink<WatchResponse>>>,
```

Comment:
In this context, we know the sender is a watch_response_sender, so we could name it "sender".